kubermatic/resource_cluster.go

```diff
@@ -453,12 +453,12 @@ func resourceClusterDelete(d *schema.ResourceData, m interface{}) error {
 			if e, ok := err.(*project.GetClusterDefault); ok && e.Code() == http.StatusNotFound {
 				k.log.Debugf("cluster '%s' has been destroyed, returned http code: %d", cID, e.Code())
 				d.SetId("")
-				return resource.RetryableError(err)
+				return nil
```

Comment:
it was in an infinite loop. Leave without error when cluster not found

Reply:
there is timeout for that https://github.com/kubermatic/terraform-provider-kubermatic/blob/master/kubermatic/resource_cluster.go#L426

so we could keep `RetryableError` in case of network problems

Reply:
ah yeah you're right, that should be `nil`

Reply:
I was waiting forever to destroy cluster